### ISSUE TITLE
Canopy is still intact with aircraft when ejected

### ISSUE THREAD

**Issue by alexthecatlover2310** [heclak/community-a4e-c#242]
When I eject from the aircraft the canopy is still intact with the aircraft 
![Screen_200807_152801](https://user-images.githubusercontent.com/69353838/89650971-7caa9d80-d8c3-11ea-91f5-aacefec94bcc.png)

**Comment by Andrew8604** [comment]
Using DCS v 55960 and A-4E-C v 1.4.2, I ejected while skidding down the runway and the canopy ended up behind the aircraft over 500 feet.  The seat was in front of and to the right, near where the pilot landed.  No canopy attached to aircraft.  I took a 23mm to the wing and was leaking fuel fast.  Flamed out on short final as I tried to lower gear.  Hydraulics were out too, no warning light...no gear. too late to think of the Emer Gear handle.  I trimmed it right above the runway, and not knowing how it behave in a belly landing (it might just explode) I ejected as the TER bomb racks were dragging on the runway.  I hadn't needed to eject after all. :)
![Bellied in A-4](https://user-images.githubusercontent.com/72777177/96081236-c1c3f080-0e6d-11eb-88f7-cee7cdd47f85.PNG)
